docs/FAQs.md

```diff
@@ -0,0 +1,27 @@
+---
+title: FAQs
+description: Frequently Asked Questions
+---
+
+
+## How can I use Powertools with Lombok?
+
+Poweretools uses `aspectj-maven-plugin` to compile-time weave (CTW) aspects into the project. In case you want to use `Lombok` or other compile-time preprocessor for your project, it is required to change `aspectj-maven-plugin` configuration to enable in-place weaving feature. Otherwise the plugin will ignore changes introduced by `Lombok` and will use `.java` files as a source. 
+
+To enable in-place weaving feature you need to use following `aspectj-maven-plugin` configuration:
+
+```xml hl_lines="5"
```

Comment:
is hl line number correct here? Seem to point to `</weaveDirectories>`

Reply:
Thanks for catching it. You are rigth - it was a bit confusing. I've changed it to highlight all 5 lines which needs to be added to configuration. I also added `<forceAjcCompile>true</forceAjcCompile>` which was not required when I tested in on my environment, but aspectj docs says that by default it's false so the code will not be recompiled. 

Please let me know of you want me to squash commits before merge.